src/Riok.Mapperly/Descriptors/MappingBuilders/EnumMappingBuilder.cs

```diff
@@ -1,134 +1,153 @@
-        {
-            ctx.ReportDiagnostic(DiagnosticDescriptors.TargetEnumValueNotMapped, member.Key, member.Value!, ctx.Target, ctx.Source);
-        }
-
-        var missingSourceValues = sourceValues.Where(field => !targetValues.ContainsValue(field.Value));
-        foreach (var member in missingSourceValues)
-        {
-            ctx.ReportDiagnostic(DiagnosticDescriptors.SourceEnumValueNotMapped, member.Key, member.Value!, ctx.Source, ctx.Target);
-        }
-
-        return new CastMapping(ctx.Source, ctx.Target);
-    }
-
-    private static TypeMapping BuildNameMapping(MappingBuilderContext ctx, bool ignoreCase)
-    {
-        var targetFieldsByName = ctx.Target.GetMembers().OfType<IFieldSymbol>().ToDictionary(x => x.Name);
-        var sourceFieldsByName = ctx.Source.GetMembers().OfType<IFieldSymbol>().ToDictionary(x => x.Name);
-
-        Func<IFieldSymbol, IFieldSymbol?> getTargetField;
-        if (ignoreCase)
-        {
-            var targetFieldsByNameIgnoreCase = targetFieldsByName
-                .DistinctBy(x => x.Key, StringComparer.OrdinalIgnoreCase)
-                .ToDictionary(x => x.Key, x => x.Value, StringComparer.OrdinalIgnoreCase);
-            getTargetField = source =>
-                targetFieldsByName.GetValueOrDefault(source.Name) ?? targetFieldsByNameIgnoreCase.GetValueOrDefault(source.Name);
-        }
-        else
-        {
-            getTargetField = source => targetFieldsByName.GetValueOrDefault(source.Name);
-        }
-
-        var enumMemberMappings = ctx.Source
-            .GetMembers()
-            .OfType<IFieldSymbol>()
-            .Select(x => (Source: x, Target: getTargetField(x)))
-            .Where(x => x.Target != null)
-            .ToDictionary(x => x.Source.Name, x => x.Target!.Name);
-
-        if (enumMemberMappings.Count == 0)
-        {
-            ctx.ReportDiagnostic(DiagnosticDescriptors.EnumNameMappingNoOverlappingValuesFound, ctx.Source, ctx.Target);
-        }
-
-        var missingSourceMembers = sourceFieldsByName.Where(field => !enumMemberMappings.ContainsKey(field.Key));
-        foreach (var member in missingSourceMembers)
-        {
-            ctx.ReportDiagnostic(
-                DiagnosticDescriptors.SourceEnumValueNotMapped,
-                member.Key,
-                member.Value.ConstantValue!,
-                ctx.Source,
-                ctx.Target
-            );
-        }
-
-        var missingTargetMembers = targetFieldsByName.Where(field => !enumMemberMappings.ContainsValue(field.Key));
-        foreach (var member in missingTargetMembers)
-        {
-            ctx.ReportDiagnostic(
-                DiagnosticDescriptors.TargetEnumValueNotMapped,
-                member.Key,
-                member.Value.ConstantValue!,
-                ctx.Target,
-                ctx.Source
-            );
-        }
-
-        return new EnumNameMapping(ctx.Source, ctx.Target, enumMemberMappings);
-    }
-}
+using Microsoft.CodeAnalysis;

```

Comment:
Lots of whitespace changes in this file, could you revert these?

Reply:
I think that file has non-Windows line-endings; but I've reverted to what was there for line-endings.